site/docs/v1/tech/troubleshooting.adoc

```diff
@@ -142,6 +143,65 @@ If you can still get a lower number of logins per second than you'd expect with
 
 If the MD5 configuration allows you to achieve a much higher logins per second, then the CPU is your bottleneck. If you are CPU bound, the only way to get more logins per second is to horizontally scale (add more nodes) or vertically scale (use larger CPUs with each node).
 
+
+== Troubleshooting Uninstall-Upgrade-Reinstall on RPMs
+
+[WARNING]
+====
+This will completely remove FusionAuth App and FusionAuth Search from your installation. +
+Proceed with caution. It is recommended you back up the database before performing any troubleshooting steps listed here.
+====
+
+=== The Nuclear Option
```

Comment:
Sorry to be a downer but this is a bit too colorful for docs. Can you make it more ... boring :) please?

Reply:
💤 🛌🏻  🙂